packages/cozy-doctypes/src/Folder.js

```diff
@@ -0,0 +1,54 @@
+const Application = require('./Application')
+const CozyFile = require('./File')
+
+/**
+ * Class representing the folder model.
+ * @extends CozyFile
+ */
+class CozyFolder extends CozyFile {
+  static async getReferencedFolders(client, reference) {
```

Comment:
No need to pass a `client`. We have access to `this.cozyClient`

Reply:
To me CozyFile and Document both are utility classes and are only exposing static methods. I did not find any constructor but I could be missing a point.

Reply:
We use https://github.com/cozy/cozy-libs/blob/master/packages/cozy-doctypes/src/Document.js#L72 to register the client in the application.

Reply:
Missed that. Thanks !